spec/sql_footprint_spec.rb

```diff
@@ -28,14 +29,14 @@
       )
     end
 
+    it 'formats selects by 1 count' do
+      expect { Widget.where(name: SecureRandom.uuid, quantity: 1).last }.to change { statements.count }.by(+1)
+    end
+
     it 'formats selects' do
       Widget.where(name: SecureRandom.uuid, quantity: 1).last
-      expect(statements.to_a).to include(
-        'SELECT  "widgets".* FROM "widgets" ' \
-        'WHERE "widgets"."name" = ? AND ' \
-        '"widgets"."quantity" = ?  ' \
-        'ORDER BY "widgets"."id" DESC LIMIT 1'
-      )
+      ap statements.to_a
+      expect(statements.to_a.to_s).to include '\"widgets\".\"name\" = ?'
```

Comment:
I think it might be better to leave this test as it was before.  Since we're testing the formatting of the output, I kinda like the thoroughness of looking for the whole statement.